core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/move.kt

```diff
@@ -146,8 +519,48 @@ public fun <T, C> MoveClause<T, C>.under(
 
 // region to
 
+/**
+ * ## Move To
+ *
+ * Moves columns, previously selected with [move] to a new position specified
+ * by [columnIndex] within the [DataFrame].
+ *
+ * Returns a new [DataFrame] with updated columns order.
+ *
+ * For more information: {@include [DocumentationUrls.Move]}
+ *
+ * ### Examples:
+ * ```
+ * df.move { age and weight }.to(0)
+ * df.move("age", "weight").to(2)
+ * ```
+ *
+ * @param [columnIndex] The index specifying the position in the [DataFrame] columns
+ *  * where the selected columns will be moved.
+ */
 public fun <T, C> MoveClause<T, C>.to(columnIndex: Int): DataFrame<T> = moveTo(columnIndex)
 
+/**
+ * ## Move ToTop
```

Comment:
Seems you are a bit inconsistent with the names of the sub-operations. Some are written like "Move ToTop", others like "MoveToStart". I assume you did this to mirror the function names, however it may be more readable to just separate them all by spaces and use Titlecase, like "Move to Top", "Move to Start", etc. Also, you'll make nikita happy if you use a couple more `#`'s so the titles become a bit smaller

Reply:
OK, so let's make a convention for all of this.

My logic was like this:
If its a complete operation (that is DF-> DF : `moveToStart`, `moveToEnd`) then it will have the full name in Upper Camel Case.
If it's a part of move operation (MoveClase -> DF), I write "Move" + this operation in UCC.

Reply:
I would favor readability over it following the function name exactly here.  Users can already see the function name when they click to read the KDoc, so the title can just be a simple readable title I think.